resources/apt/apt.dsc.resource.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+export exist=true
+export NONINTERACTIVE=1
+
+# $packageName and $_exist are sent as env vars by dsc converting the JSON input to name/value pairs
+
+check_args() {
+    if [[ -z $packageName ]]; then
+        echo "packageName not set"
+        exit 1
+    fi
+}
+
+get_apt() {
+    pkgname=$1
+    InstalledSection=0
+    apt list --installed $pkgname 2>&1 | while read line; do
+        if [[ $line == Listing* ]]; then
+            InstalledSection=1
+        elif [[ $InstalledSection = 1 ]]; then
+            echo $line | awk '{
+                split($0, a, " ");
+		split(a[1], pn, "/");
+                printf("{ \"_exist\": \"%s\", \"packageName\": \"%s\", \"Version\": \"%s\", \"Source\": \"%s\" }\n", ENVIRON["exist"], pn[1], a[2], pn[2]);
+            }'
+        fi
+    done
+}
+
+if [[ "$#" -eq "0" ]]; then
+    echo "Command not provided, valid commands: get, set, export"
+    exit 1
+elif [[ "$1" == "get" ]]; then
+    check_args
+    output="$(get_apt $packageName)"
+    if [[ -z $output ]]; then
+	    printf '{ "_exist": "false", "packageName": "%s", "Version": "", "Source": "" }\n' $packageName
```

Comment:
We should always return a compressed JSON Line from our resources.

If we're going to keep the version and source fields, the casing should match the schema:
```suggestion
	    printf '{"_exist":"false","packageName":"%s","version":"","source":""}\n' $packageName
```
However, because both version and source are non-required, we can simply omit them:
```suggestion
	    printf '{"_exist":"false","packageName":"%s"}\n' $packageName
```